app/controllers/items_controller.rb

```diff
@@ -21,6 +21,6 @@ def destroy
     @list = List.find(params[:list_id])
```

Comment:
`@list = List.find(params[:list_id])` is a potential security issue if a logged in attacker tries requesting various list_ids and ids. Try accessing the list via `current_user` as done elsewhere in the controller.